src/compareImage.test.js

```diff
@@ -109,7 +98,6 @@ describe('Compare Image', () => {
   });
 
   it('writes to disk diff image if saveDifferencifiedImage is true', async () => {
-    expect.assertions(1);
```

Comment:
why remove this?

Reply:
Mainly because compare image won't throw anymore. But yeah, we can keep it with `expect.assertion(0)`

Reply:
Right, yeah, it now logs errors rather than throw. Should the tests not be changed to reflect that?